index.js

```diff
@@ -1,21 +1,12 @@
-/* eslint-disable linebreak-style */
-/* eslint-disable no-unused-vars */
-/* eslint-disable import/no-unresolved */
-/* eslint-disable linebreak-style */
-/* eslint-disable no-trailing-spaces */
-/* eslint-disable linebreak-style */
 /* eslint-disable func-names */
-/* eslint-disable semi */
-/* eslint-disable linebreak-style */
 /* eslint-disable import/extensions */
```

Comment:
- [ ] As suggested by the previous reviewer, please make sure to remove any `eslint-disable` as that defies the purpose of using the linters in the first place.